spring-boot-project/spring-boot-test/src/main/java/org/springframework/boot/test/context/SpringBootConfigurationFinder.java

```diff
@@ -33,20 +34,27 @@
  *
```

Comment:
That part of the Javadoc should be rephrased.

Reply:
done